src/server.ts

```diff
@@ -14,6 +50,23 @@ const resolvers = {
       return 'hello, world!';
     },
   },
+
+  Mutation: {
+    login: (_, { login }) => {
+      const { email, password } = login;
+
+      const user = fakeUsers.filter((user) => user.email === email && user.password === password)[0];
+      const token = 'Token';
+
+      if (!user) {
+        throw new Error('Unauthorized. Possible invalid credentials.');
+      }
+
+      const response = { user, token };
+
+      return response;
```

Comment:
Aqui pode retornar direto né?
```suggestion
      return { user, token };
```